docs(k8s): updated version support policy page (#2350)
scaleway/docs-content: containers/kubernetes/reference-content/version-support-policy.mdx

</Message>

### Cluster auto-upgrade

Before using the auto-upgrade feature, it is essential to differentiate between patch upgrades and minor version upgrades in Kubernetes. While a patch upgrade typically focuses on bug fixes and security patches without altering existing functionalities, a minor version upgrade introduces new features and changes.

> The [Kubernetes Kapsule](https://www.scaleway.com/en/kubernetes-kapsule/) and [Kubernetes Kosmos](https://www.scaleway.com/en/kubernetes-kosmos/) platforms offer an auto-upgrade feature that specifically addresses to patch upgrades within the same minor Kubernetes version you are currently using. This means that the feature allows for updates to the latest patch, ensuring enhanced security and stability without introducing major changes.

Reviewer: ```suggestion
The [Kubernetes Kapsule](https://www.scaleway.com/en/kubernetes-kapsule/) and [Kubernetes Kosmos](https://www.scaleway.com/en/kubernetes-kosmos/) platforms offer an auto-upgrade feature to address patch upgrades within the same minor Kubernetes version you are currently using. This means that the feature allows for updates to the latest patch, ensuring enhanced security and stability without introducing major changes.
```

This upgrade applies to both the control plane and the managed node versions of Kubernetes. You can activate this feature through the API, CLI, or the Scaleway console. Note, to ensure smooth operation of the auto-upgrade, the `kube-system` namespace should remain untouched and free from any user-initiated modifications that might cause issues with the upgrade process.

> If you are looking to upgrade to the next minor version of Kubernetes, refer to our [How To guide for upgrading to the next minor version](https://www.scaleway.com/en/docs/containers/kubernetes/how-to/upgrade-kubernetes-version/).

Reviewer: ```suggestion
If you are looking to upgrade to the next minor version of Kubernetes, refer to our [documentation on upgrading to the next minor version](/containers/kubernetes/how-to/upgrade-kubernetes-version/).
```
